libraries/CryptoInterface/examples/HelloCrypto/HelloCrypto.ino

```diff
@@ -0,0 +1,102 @@
+/**
+   This example shows the functionality of the CryptoInterface library.
+*/
+
+#include <ESP8266WiFi.h>
+#include <TypeConversion.h>
+#include <CryptoInterface.h>
+
+namespace TypeCast = esp8266::TypeConversion;
+using namespace experimental;
+
+/**
+   NOTE: Although we could define the strings below as normal String variables,
+   here we are using PROGMEM combined with the FPSTR() macro (and also just the F() macro further down in the file).
+   The reason is that this approach will place the strings in flash memory which will help save RAM during program execution.
+   Reading strings from flash will be slower than reading them from RAM,
+   but this will be a negligible difference when printing them to Serial.
+
+   More on F(), FPSTR() and PROGMEM:
+   https://github.com/esp8266/Arduino/issues/1143
+   https://arduino-esp8266.readthedocs.io/en/latest/PROGMEM.html
+*/
+constexpr char masterKey[] PROGMEM = "w86vn@rpfA O+S"; // Use 8 random characters or more
+
+void setup() {
+  // Prevents the flash memory from being worn out, see: https://github.com/esp8266/Arduino/issues/1054 .
+  // This will however delay node WiFi start-up by about 700 ms. The delay is 900 ms if we otherwise would have stored the WiFi network we want to connect to.
+  WiFi.persistent(false);
+
+  Serial.begin(115200);
+  delay(50); // Wait for Serial.
+
+  //yield(); // Use this if you don't want to wait for Serial.
+
+  // The WiFi.disconnect() ensures that the WiFi is working correctly. If this is not done before receiving WiFi connections,
+  // those WiFi connections will take a long time to make or sometimes will not work at all.
```

Comment:
@d-a-v, I'm not sure I agree with the comment.  Can you please chime in?

Reply:
Seems like I added this in a commit 2018-09-14, and back then I know it was required. I can't see any problem now when removing it in the example code, so maybe this was a bug that has been fixed. With the caveat that my original test case may have been different, it would seem as though `WiFi.disconnect()` can be removed here.

Reply:
Yes, please.  If we do have a problem we should fix it in the core and not workaround it in this example.